src/app/packages/open-layer-visualizers/entities-visualizer.ts

```diff
@@ -61,13 +68,14 @@ export class EntitiesVisualizer implements IMapVisualizer {
 		this._footprintsVector = new VectorLayer({
 			source: this._source,
 			style: this.featureStyle.bind(this),
-			opacity: this.containerLayerOpacity
+			opacity: this.containerLayerOpacity,
+			zIndex: 50000
 		});
-
+		this._footprintsVector.setZIndex(10000)
```

Comment:
Why both? 
zIndex: 50000
this._footprintsVector.setZIndex(10000)

missing ';'